utils/route.helpers.js

```diff
@@ -1,134 +1,158 @@
-const { checkSchema } = require('express-validator')
-const { routes: defaultRoutes } = require('../config/routes.config')
-const { checkErrors } = require('./validate.helpers')
+const { checkSchema } = require("express-validator");
+const { routes: defaultRoutes } = require("../config/routes.config");
+const { checkErrors } = require("./validate.helpers");
+const url = require("url");
 
-const DefaultRouteObj = { name: false, path: false }
+const DefaultRouteObj = { name: false, path: false };
 
 /**
  * This request middleware checks if we are visiting a public path
  */
 const checkPublic = function(req, res, next) {
-  const publicPaths = ['/', '/clear', '/start']
+  const publicPaths = ["/", "/clear", "/start"];
   if (publicPaths.includes(req.path)) {
-    return next()
+    return next();
   }
 
-  return next()
-}
+  return next();
+};
 
 const routeHasIndex = route => {
-  if (!route || !route.hasOwnProperty('index')) {
-    return false
+  if (!route || !route.hasOwnProperty("index")) {
```

Comment:
There seems to be a war between single and double quotes going on

Reply:
oh wait, my vscode at home was changing the single quotes to double quotes? That's odd...